manifests/vhost.pp

```diff
@@ -1913,6 +1916,7 @@
   Optional[String] $modsec_body_limit                                                 = undef,
   Optional[Integer[1, default]] $modsec_inbound_anomaly_threshold                     = undef,
   Optional[Integer[1, default]] $modsec_outbound_anomaly_threshold                    = undef,
+  Optional[String] $modsec_allowed_methods                                            = undef,
```

Comment:
@Vincevrp
Could you give me your reasoning on why this is a String and not and Array?

Reply:
@david22swan To be consistent with https://github.com/puppetlabs/puppetlabs-apache/blob/main/manifests/mod/security.pp#L124